fix(iam): remove own user MTA-3148 (#1613)
scaleway/docs-content: identity-and-access-management/iam/how-to/manage-users.mdx

## How to view and delete user API keys

1. From the user's [Overview tab](#how-to-access-the-user-overview), click the **Credentials** tab.
    You can view details of the user's [API keys](/identity-and-access-management/iam/how-to/create-api-keys/).
    <Lightbox src="scaleway-user-api-keys.webp" alt="" />
    <Message type="note">
      You cannot see the secret key part of any API keys, nor can you create an API key for any IAM user other than yourself.
    </Message>
    If you want to delete a user's API key, proceed to the following steps:
2. Click <Icon name="more" /> next to the API key you wish to delete, and select **Delete**.
    A warning displays, reminding you that this action is permanent.
3. Type **DELETE** and click **Submit** to confirm the action.

## How to remove a user from the Organization

<Message type="note">

>   To remove all users other than yourself you need to either be the [Owner](/identity-and-access-management/iam/concepts#owner) of the Organization, or have [IAMManager permissions](/identity-and-access-management/iam/reference-content/permission-sets). The exception to this is that you can always remove your own, personal IAM user from an Organization in which you are a [Guest](/identity-and-access-management/iam/concepts#guest), anytime. You cannot remove your IAM user from an Organization you own.

Reviewer: ```suggestion
  To remove any users other than yourself you need to either be the [Owner](/identity-and-access-management/iam/concepts#owner) of the Organization, or have [IAMManager permissions](/identity-and-access-management/iam/reference-content/permission-sets). The exception to this is that you can always remove yourself as an IAM user from an Organization in which you are a [Guest](/identity-and-access-management/iam/concepts#guest), anytime. This equates to leaving the Organization. You cannot leave an Organization that you own.
```
